buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -365,11 +365,10 @@ class BeamModulePlugin implements Plugin<Project> {
     def cassandra_driver_version = "3.8.0"
     def classgraph_version = "4.8.56"
     def gax_version = "1.52.0"
-    def generated_grpc_beta_version = "0.44.0"
     def generated_grpc_ga_version = "1.83.0"
-    def generated_grpc_dc_beta_version = "0.27.0-alpha"
+    def generated_grpc_dc_beta_version = "0.29.0-alpha"
```

Comment:
This is not the latest but 0.30.0-alpha requires code change. Stopping at 0.29.0-alpha for simplicity.